src/hooks/useGetAllReleases.ts

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { PipelineRunLabel } from '../consts/pipelinerun';
+import { K8sQueryListResourceItems } from '../k8s';
+import { ReleaseModel } from '../models';
+import { ReleaseKind } from '../types';
+
+export interface Release {
```

Comment:
I don't remember if I asked it before, but couldn't we use `ReleaseKind` type instead of defining a new interface?